.tool-versions

```diff
@@ -1,2 +1,2 @@
-nodejs 16.0.0
+nodejs 16.10.0
```

Comment:
It will be good to test and make sure this works in stage in the deployed environment. I don't imagine it will have issues though since it is just an S3 bucket.